packages/mdc-dialog/index.ts

```diff
@@ -22,66 +22,17 @@
  */
 
 import MDCComponent from '@material/base/component';
+import {closest, matches} from '@material/dom/ponyfill';
 import {MDCRipple} from '@material/ripple/index';
-
+import * as createFocusTrap from 'focus-trap';
```

Comment:
Use two separate imports: one for the function, and one for the type definitions:

```ts
import createFocusTrap from 'focus-trap';
import * as FocusTrapLib from 'focus-trap';
```

Reply:
Are you suggesting we shouldn't need to re-export FocusTrapLib from our own `./types` (which is where we're currently importing it from)?

Reply:
Turns out we need to do it this way:

```ts
import * as createFocusTrap from 'focus-trap';

// Function signature
createFocusTrap(...);

// Interface definitions
createFocusTrap.FocusTrap;
createFocusTrap.Options;
```

Although Focus Trap's `.d.ts` file claims to have a default export, default imports don't work:

```ts
import createFocusTrap from 'focus-trap';

// ERROR: undefined is not a function
createFocusTrap(...);
```